SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -99,15 +99,24 @@ struct FConnectionConfig
 		if (LinkProtocolString.Compare(TEXT("Tcp"), ESearchCase::IgnoreCase) == 0)
 		{
 			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_TCP;
+			return;
 		}
 		else if (LinkProtocolString.Compare(TEXT("Kcp"), ESearchCase::IgnoreCase) == 0)
 		{
 			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_KCP;
+			return;
 		}
-		else if (!LinkProtocolString.IsEmpty())
+
+		// None given or not recognised, use default (for server this is TCP, for client this is KCP)
+		if (SpatialConstants::DefaultServerWorkerType.ToString().Equals(WorkerType))
+		{
+			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_TCP;
+		}
```

Comment:
Doesn't seem like we explicitly default to KCP anywhere, is that in the constructor - should we be explicit in this method?

Reply:
It's in the constructor, but explicitly setting it is a good idea so I'll add that

Reply:
Actually. I think that the better solution is to default to TCP and then switch to KCP when you detect a ClientWorker type. My reasoning here is that in the future we are intending to bring back different server WorkerTypes. This would mean this switch would need to be configured for any users custom worker types such as PhysicsWorker and OffloadWorker to think of a couple.

Since we only ever want clients to use KCP then swapping this logic makes more sense.